music_assistant/server/providers/soundcloud/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "music",
+  "domain": "soundcloud",
+  "name": "Soundcloud",
+  "description": "Support for the Soundcloud streaming provider in Music Assistant.",
+  "codeowners": ["@gieljnssns"],
+  "config_entries": [
+    {
+      "key": "client_id",
+      "type": "secure_string",
+      "label": "Client id"
+    },
+    {
+      "key": "authorization",
+      "type": "secure_string",
+      "label": "Authorization"
+    }
+  ],
+
+  "requirements": ["bs4==0.0.1"],
+  "documentation": "",
```

Comment:
can we create a link to the documentation here ?

Reply:
Where should the documentation go to?
A .md file?

Reply:
https://github.com/orgs/music-assistant/discussions/1160

Reply:
I have created the initial page and invite you to the members so you can edit it yourself.